README.md

```diff
@@ -64,32 +64,28 @@ Please read the Requirements section. Thank you!
 7. Select the volume containing the bootable installer. Then click the up arrow or press Return. 
 8. Choose your language, if prompted.
 9. Open the Terminal.
-10. Execute `cd /Volumes/bigmac; .preinstall.sh`, and quit Terminal.
+10. Execute `/Volumes/bigmac/preinstall.sh`, and quit Terminal.
 11. Select Install macOS Big Sur from the Utilities window, then click Continue and follow the onscreen instructions.
 
-Legacy USB 2.0 for keyboard and mouse devices is not working 100%. If you leave them plugged into a USB 2.0 port, they will work. If you disconnect them, they won't reconnect. This is a problem with Big Sur which Apple introduced on their own. A workaround is use a USB3.0 PCIe card but this only works when the system is booted, not from a cold start. USB 3.0 hubs plugged into a USB 2.0 port may also work. A patch may be implemented to take care of this issue.
+Legacy USB 2.0 for keyboard and mouse devices is never working. If you leave them plugged into a USB 2.0 port, they will work. If you disconnect them, they won't reconnect. This is a problem with Big Sur which Apple introduced on their own. A workaround is use a USB3.0 PCIe card but this only works when the system is booted, not from a cold start. USB 3.0 hubs plugged into a USB 2.0 port may also work. A patch may be implemented to take care of this issue.
 
-Note: Option boot using a boot screen requires a keyboard in directly into the machine and not a PCIe card. Bluebooth keyboards do not catch up in time and PCIe cards don't work without an OS. There used to be a a Bluetooth fix in NVRAM for this. If I locate the fix, will update.
+Note: Option boot using a boot screen requires a keyboard in directly into the machine and not a PCIe card. Bluebooth keyboards do not catch up in time and PCIe cards don't work without an OS. There used to be a a Bluetooth fix in NVRAM for this. If I locate the fix, I will update.
 
 ## BigMac's Workflow in a Nut Shell 🥜
-1. Workflow -> Download Big Sur -> Create USB Installer
+1. Workflow -> [Download Big Sur](http://swcdn.apple.com/content/downloads/00/55/001-86606-A_9SF1TL01U7/5duug9lar1gypwunjfl96dza0upa854qgg/InstallAssistant.pkg) -> Create USB Installer (`sudo /Applications/Install\ macOS\ Big\ Sur.app/Contents/Resources/createinstallmedia --volume /Volumes/MyVolume`)
```

Comment:
BigMac.sh already handles the download. The link should not be in a public facing page. This violates Apple's policy and can cause BigMac to get shutdown.